src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -508,6 +508,10 @@ void drawing_classifier::iterate_training() {
         {"validation_accuracy", average_val_accuracy},
         {"validation_log_loss", average_val_loss},
     });
+  } else {
+    add_or_update_state({
+      {"validation_accuracy", FLEX_UNDEFINED}
```

Comment:
We should add `validation_log_loss` hear ideally.

Reply:
Done, thanks!